package Test;

import static org.junit.Assert.*;

public class PartidaTest {
    @org.junit.Before
    public void setUp() throws Exception {
    }

    @org.junit.After
    public void tearDown() throws Exception {
    }


    @org.junit.Test
    public void getId() throws Exception {
    }

    @org.junit.Test
    public void setId() throws Exception {
    }

    @org.junit.Test
    public void getTiempo() throws Exception {
    }

    @org.junit.Test
    public void setTiempo() throws Exception {
    }

    @org.junit.Test
    public void isRolMaker() throws Exception {
    }

    @org.junit.Test
    public void setRolMaker() throws Exception {
    }

    @org.junit.Test
    public void getDificultad() throws Exception {
    }

    @org.junit.Test
    public void setDificultad() throws Exception {
    }

    @org.junit.Test
    public void getNumColores() throws Exception {
    }

    @org.junit.Test
    public void setNumColores() throws Exception {
    }

    @org.junit.Test
    public void getNumColumnas() throws Exception {
    }

    @org.junit.Test
    public void setNumColumnas() throws Exception {
    }

    @org.junit.Test
    public void isGanado() throws Exception {
    }

    @org.junit.Test
    public void sumaTiempo() throws Exception {
    }

    @org.junit.Test
    public void setCodigoSecreto() throws Exception {
    }

    @org.junit.Test
    public void getCodigoSecreto() throws Exception {
    }

    @org.junit.Test
    public void getNumeroFilaActual() throws Exception {
    }

    @org.junit.Test
    public void generaCodigoSecretoAleatorio() throws Exception {
    }

    @org.junit.Test
    public void getSiguienteIntento() throws Exception {
    }

    @org.junit.Test
    public void generaPuntuacion() throws Exception {
    }

    @org.junit.Test
    public void imprimeInfo() throws Exception {
    }

    @org.junit.Test
    public void imprimeAllInfo() throws Exception {
    }

    @org.junit.Test
    public void generaRespuesta() throws Exception {
    }

    @org.junit.Test
    public void setIntento() throws Exception {
    }

    @org.junit.Test
    public void setRespuesta() throws Exception {
    }

    @org.junit.Test
    public void getUltimaRespuesta() throws Exception {
    }

    @org.junit.Test
    public void getPista1() throws Exception {
    }

    @org.junit.Test
    public void getPista2() throws Exception {
    }

    @org.junit.Test
    public void getPista3() throws Exception {
    }

}